crates/storage_models/src/enums.rs

```diff
@@ -6,11 +6,12 @@ pub mod diesel_exports {
         DbEventClass as EventClass, DbEventObjectType as EventObjectType, DbEventType as EventType,
         DbFutureUsage as FutureUsage, DbIntentStatus as IntentStatus,
         DbMandateStatus as MandateStatus, DbMandateType as MandateType,
-        DbMerchantStorageScheme as MerchantStorageScheme, DbPaymentFlow as PaymentFlow,
+        DbMerchantStorageScheme as MerchantStorageScheme, DbPaymentExperience as PaymentExperience,
+        DbPaymentFlow as PaymentFlow, DbPaymentIssuer as PaymentIssuer,
         DbPaymentMethodIssuerCode as PaymentMethodIssuerCode,
         DbPaymentMethodSubType as PaymentMethodSubType, DbPaymentMethodType as PaymentMethodType,
         DbProcessTrackerStatus as ProcessTrackerStatus, DbRefundStatus as RefundStatus,
-        DbRefundType as RefundType, DbRoutingAlgorithm as RoutingAlgorithm,
```

Comment:
If `RoutingAlgorithm` enum is no longer being used, please remove it from `enums.rs` and add/update migrations to remove the database type.

Reply:
Removed the enum. The database type has already been removed in #362.